mycroft/client/speech/hotword_factory.py

```diff
@@ -142,16 +143,17 @@ def get_model_info(self):
         return model_name, model_path
 
     def find_download_exe(self):
-        exe_file = resolve_resource_file(self.exe_name)
-        if exe_file:
-            return exe_file
         try:
-            if call(self.exe_name + ' < /dev/null', shell=True) == 0:
+            if call(['command', '-v', self.exe_name], stdout=PIPE) == 0:
```

Comment:
Will this ever work without `shell=True`? `command` is not an application as far as I can read but implemented in the shell?

```python
>>> from subprocess import call, PIPE
>>> call(['command', '-v', 'bash'], stdout=PIPE)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/subprocess.py", line 523, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1343, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
>>> call(['command', '-v', 'bash'], shell=True, stdout=PIPE)
0
>>>
```

Reply:
Nice catch. Fixed.